src/runtime/runtime.cs

```diff
@@ -80,16 +80,22 @@ public static IntPtr GetProcAddress(IntPtr dllHandle, string name)
     /// </summary>
     public class Runtime
     {
+        // C# compiler copies constants to the assemblies that references this library.
+        // We needs to replace all public constants to static readonly fields to allow 
+        // binary substitution of different Python.Runtime.dll builds in a target application.
+
+        public static readonly int UCS = _UCS;
```

Comment:
If we ever want to have a single DLL for all platforms and Python versions, the values of `pyversion` or `UCS` will have to be evaluated at runtime on startup, that's what I meant.